draft-thomson-webpush-protocol.xml

```diff
@@ -655,19 +586,18 @@ DATA         [stream 4] +END_STREAM
           include the "private" directive.
         </t>
         <t>
-          A push service can invalidate a registration or subscription at any
-          time.  If a user agent has an outstanding request to the
-          "registration" resource (see <xref target="monitor"/>), this can be
-          signaled by returning a 400-series status code, such as 410 (Gone).  A
-          push service uses server push to indicate that a subscription has
-          expired; a pushed 400-series status code for the subscription resource
-          signals the termination of a subscription.
+          A push service can invalidate a subscription at any time. If a user
+          agent has an outstanding request to the "monitor" resource (see <xref
+          target="monitor"/>), this can be signaled by returning a 400-series
+          status code, such as 410 (Gone).  A push service uses server push to
+          indicate that a subscription has expired; a pushed 400-series status
+          code for the "monitor" resource signals the termination of a
```

Comment:
We don't need to push a response for the monitor now that we are talking to it directly.  Just have the long poll return the 400 directly.